cicecore/cicedynB/dynamics/ice_dyn_shared.F90

```diff
@@ -176,7 +176,7 @@ subroutine init_dyn (dt)
          if (trim(coriolis) == 'constant') then
             fcor_blk(i,j,iblk) = 1.46e-4_dbl_kind ! Hibler 1979, N. Hem; 1/s
          else if (trim(coriolis) == 'zero') then
-            fcor_blk(i,j,iblk) = 0.0
+            fcor_blk(i,j,iblk) = c0
```

Comment:
Is the change in this line the entire reason the code is not BFB for cor=0?

Reply:
yes that is why I put in the comment. I dont think that I reran a test with coriolis=0.0 after the last commit but just a warning if the standard test complains about bit for bit differences. 
All other test have been bit for bit with main after Tony's OMP updates.
Other than that all test are bit for bit.